src/cargo/ops/cargo_doc.rs

```diff
@@ -47,7 +47,11 @@ pub fn doc(ws: &Workspace<'_>, options: &DocOptions) -> CargoResult<()> {
                     .join("index.html");
                 if path.exists() {
                     let mut shell = ws.config().shell();
-                    shell.status("Generated", path.display())?;
+                    let link = shell.err_file_hyperlink(&path);
+                    shell.status(
+                        "Generated",
```

Comment:
Need to add this to `--timings` as well

Might also add it to the above `Opening`